### ISSUE TITLE
Importing images doesn't make much sense in a non-Webmaker context

### ISSUE THREAD

**Issue by toolness** [mozilla/webmaker-addons#57]
If we're not creating this add-on as a bridge to Webmaker, there are some design decisions that retrospectively don't make a lot of sense in the new context.

One example of this is the fact that only existing images in `<img>`/`<image>`/`background-image:` can be imported into the canvas. We're creating a making experience that is limiting itself to the implementation structure of a web page, while the user will quite likely want to take anything they can  see and bring it into their composition space, regardless of whether it happens to be referenced as a URL in the page's source code.

A better affordance would likely be something like the one offered by tools like [Snag It](http://www.techsmith.com/snagit-features.html): a rectangle could "snap" to images and divs in the page, but users should ultimately be able to capture whatever they want and bring it into the tool.

Furthermore, if we don't care about exporting to HTML (or at least, making any exported HTML readable/hackable) then we're also free to represent the user's composition in any way we want. This was previously a constraint in the webmaker-focused version of the add-on.

**Comment by xmatthewx** [comment]
I agree with some of the things you said, but not with the title of this issue. Many people want to extract an image asset from a page. Importing divs doesn't make much sense in a non-Webmaker context. 

I think that switching from images to html or screenshots solves edge cases while introducing lots of complexity to the most common user need (grabbing an image to reshare). It solves a similar, but different problem. 

I like the idea of grabbing a region of the screen by snapping rather than the usual selecting. I think this ties a user more deeply to the implementation structure of a web page, rather than liberating the assets. It's not mvp.  

Full image resolution, images behind curtains, icons with transparent backgrounds -- these things become more tricky to extract from the DOM with a hover, impossible with a tap. 

There are several solid tools for screen captures. What's wrong with them? There currently isn't a simple means for people to grab, edit, and reshare images without touching the filesystem.

**Comment by xmatthewx** [comment]
p.s. I think this is an interesting idea for a sibling add-on. And if it proves to be an easier way to extract images without confusion, then huzzah. 

I think the current addon needs to be even less complex than it's current implementation to be truly useful to the broadest audience possible.

**Comment by davidascher** [comment]
What this addon does today very well that screen grabbers can't is find the
image behind the text.
